tests/ui/floating_point_powf.rs

```diff
@@ -18,6 +18,9 @@ fn main() {
     let _ = x.powf(-16_777_215.0);
     let _ = (x as f32).powf(-16_777_215.0);
     let _ = (x as f32).powf(3.0);
+    let _ = (1.5_f32 + 1.0).powf(1.0 / 3.0);
+    let _ = 1.5_f64.powf(1.0 / 3.0);
```

Comment:
These tests is for `imprecise_flops`. Can the tests for `suboptimal_flops` be added? 
```suggestion
    let _ = 1.5_f64.powf(1.0 / 3.0);
    let _ = 1.5_f64.powf(1.0 / 2.0);
    let _ = 1.5_f64.powf(3.0);
```